spec/index.html

```diff
@@ -495,6 +496,26 @@ <h2>RDF Documents and Syntaxes</h2>
     and still the subject of group discussion. An alternative to conformance
     levels, "profiles", may be adopted instead, abandoned, or described in 
     another specification.</p>
+
+  <section id="rdf-strings">
+    <h2>Strings in RDF</h2>
+
+    <p>RDF uses Unicode [[Unicode]] as the fundamental representation for string values.
+      Within this, and related specifications, the term <dfn id="dfn-rdf-string">string</dfn>
+      is used to describe an ordered sequence of zero or more <a data-lt="code point" class="lint-ignore">Unicode code points</a>
```

Comment:
Because the lexical forms of RDF literals do not contain surrogate code points, neither do RDF strings.
```suggestion
      is used to describe an ordered sequence of zero or more <a data-lt="code point" class="lint-ignore">Unicode code points</a> matching the <a data-cite="XML11#charsets">Char</a> production [[XML11]], and thus do not contain Unicode surrogate code points.
```

Reply:
tweaking @pfps's language
```suggestion
      is used to describe an ordered sequence of zero or more <a data-lt="code point" class="lint-ignore">Unicode code points</a> which matches the <a data-cite="XML11#charsets">Char</a> production of [[XML11]], and thus does not contain Unicode surrogate code points.
```

Reply:
I believe @afs is objecting to the use of the Char production in RDF Concepts, and limiting it to just describe a sequence of zero or more Unicode code points without further restriction.

Reply:
```suggestion
is used to describe an ordered sequence of zero or more <a data-lt="code point" class="lint-ignore">Unicode code points</a> which are <a data-cite="i18n-glossary#dfn-scalar-value" class="lint-ignore">Unicode scalar values</a>, and thus does not contain Unicode surrogate code points.
```

Reply:
Once more into the breach...
```suggestion
      is used to describe an ordered sequence of zero or more <a data-lt="code point" class="lint-ignore">Unicode code points</a> which are <a data-cite="i18n-glossary#dfn-scalar-value" class="lint-ignore">Unicode scalar values</a>, which do not include Unicode surrogate code points.
```